presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryCompletedEvent.java

```diff
@@ -45,6 +48,7 @@ public QueryCompletedEvent(
         this.context = requireNonNull(context, "context is null");
         this.ioMetadata = requireNonNull(ioMetadata, "ioMetadata is null");
         this.failureInfo = requireNonNull(failureInfo, "failureInfo is null");
+        this.warnings = requireNonNull(warnings, "warnings is null");
```

Comment:
We don't want dependency on guava here. QueryInputMetadata doesn't do copyOf as well.